modules/test-utils/src/index.ts

```diff
@@ -11,6 +11,7 @@ export {PerformanceTestRunner} from './performance-test-runner';
 export {webglDevice, webgpuDevice} from './create-test-device';
```

Comment:
Are we going to export a precreated `nullDevice`?
Are we going to add a `NullDevice` to `getTestDevices()`